File: platforms/swift/examples/geometry/mesh/add_instances.swift
import FragmentColor

let m = Mesh()
let red = [1.0, 0.0, 0.0, 1.0]
let green = [0.0, 1.0, 0.0, 1.0]
let blue = [0.0, 0.0, 1.0, 1.0]
m.addInstances([
    try Instance().set("tint", red),
    try Instance().set("tint", green),
    try Instance().set("tint", blue),
])

File: platforms/python/examples/geometry/mesh/add_instances.py
from fragmentcolor import Mesh, Instance

m = Mesh()
red = [1.0, 0.0, 0.0, 1.0]
green = [0.0, 1.0, 0.0, 1.0]
blue = [0.0, 0.0, 1.0, 1.0]
m.add_instances([
    Instance().set("tint", red),
    Instance().set("tint", green),
    Instance().set("tint", blue),
])